includes/Admin/ReviewNotice.php

```diff
@@ -0,0 +1,153 @@
+                    It would motivate us a lot.', 'wemail'
+                );
+				?>
+                </p>
+                <div class="wemail-reivew-notice-connect-button" style="margin-bottom: 20px">
+                    <form action='' method='post'>
+                        <button type="submit" class="button review_reposnse_yes" name="review_reposnse_yes">
+                                Yes, Absolutely
+                        </button>
+                        <button type="submit" class="response-button" name="review_response_later">  <u> Ask me later </u> </button>
+                        <button type="submit" class="response-button" name="review_response_no"> <u> No, not good enough </u> </button>
+                    </form>
+                </div>
+            </div>
+        </div>
+        <?php
+    }
+
+    /**
+     * Show notice to connect site in weMail
+     *
+     * @return bool
+     */
+    public function connect_review_notice() {         if ( isset( $_GET['dismiss_wemail_review_notice'] ) && (int) $_GET['dismiss_wemail_review_notice'] === 1 ) {
+            update_option( 'wemail_review_notice', 1 );
+	}
+
+        $installed_time = get_option( 'wemail_installed_time' );
+
+        $this->handleReviewResponse();
+
+        $this->checkCampaign();
+
+	if ( (int) get_option( 'wemail_review_notice' ) !== 1 ) {
+        $this->time_based_review = ( time() - $installed_time ) / 86400 > 7;
+        $this->day_count = (int) ( time() - $installed_time ) / 86400;
+        $this->campaign_count = (int) get_option( 'wemail_sent_campaign_count' );
+        $campaign_based_review = (int) get_option( 'wemail_sent_campaign_count' ) >= 3;
+        if ( $this->time_based_review || $campaign_based_review ) {
+			add_action( 'admin_notices', [ $this, 'connect_review_notice_html' ] );
+		}
+	}
+    }
+
+    public function review_response( $response ) {
+        update_option( 'wemail_response_for_review_notice', $response );
+        update_option( 'wemail_review_notice', 1 );
+    }
+
+    /**
+     * @return void
+     */
+    public function checkCampaign() {
+        $campaigns = wemail()->api->campaigns()->get();
+            if ( is_array($campaigns) && !empty($campaigns['data']) && (int) get_option( 'wemail_sent_campaign_count' ) < 3 && get_transient( 'wemail_sent_campaign_count' ) === false ) {
+            $count_campaigns = count( $campaigns['data'] );
+            set_transient( 'wemail_sent_campaign_count', $count_campaigns, 60 * 60 * 2 );
+            update_option( 'wemail_sent_campaign_count', $count_campaigns );
+		}
+    }
+
+    /**
+     * @return void
+     */
+    public function handleReviewResponse() {
+        // phpcs:ignore
+        if ( isset( $_POST['review_reposnse_yes'] ) ) {
+            $this->review_response( 'yes' );
+            header( 'Location:https://wordpress.org/support/plugin/wemail/reviews/#new-post' );
+		}
+        // phpcs:ignore
+		if ( isset( $_POST['review_response_later'] ) ) {
```

Comment:
Also, you can store the user response in a single key. So that you can check it via `switch case`.